.github/workflows/main.yml

```diff
@@ -399,7 +417,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        wallet: ["vdrtools_wallet", "askar_wallet"]
+        wallet: ["main_vdrtools_wallet,target_askar_wallet", "main_askar_wallet,target_askar_wallet"]
```

Comment:
Integration test do not know about migrations, do they? I am sure there was technical reason which required you introduce these feature flags here, but conceptually when running normal integration tests, there should be no notion of migrations.

Reply:
Well, libvcx [knows](https://github.com/hyperledger/aries-vcx/pull/1144/files#diff-67003c7e81812e12857fcefa42000b0b3d953a8db1073404a173d7bd9888682fR275) about migrations and exposes the migration as a part of its public API.

The reason for the new feature flags is that a migration has source and destination wallet and we need to decide which wallet is to be considered 'main' so that the appropriate global variable is included. Anyway, this seems to be irrelevant now, removing the vcx_napi_rs and libvcx_core will definitely make things easier.